webpack.config.dev.js

```diff
@@ -23,8 +30,8 @@ export default {
   ],
   module: {
```

Comment:
According to the webpack migration docs, loaders have been replaced by rules in modules, among other changes...
module: {
        rules: [{
            test: /\.js$/,
            exclude: /node_modules/,
            loader: 'babel-loader'
        }, {
            test: /(\.css|\.scss|\.sass)$/,
            exclude: /node_modules/,
            use: ExtractTextPlugin.extract({
                fallback: "style-loader",
                use: [{
                    loader: 'css-loader',
                    options: {
                        minimize: true,
                        sourceMap: true
                    }
                }, {
                    loader: 'sass-loader',
                    options: {
                        includePaths: [path.resolve(__dirname, 'src', 'scss')],
                        sourceMap: true
                    }
                }]
            })
        }]
    }